sqrt_decomposition/sqrt_decomposition_simple.go

```diff
@@ -0,0 +1,105 @@
+package sqrt_decomposition
```

Comment:
This is a data structure, please put this algorithm inside the `structure` package.